FirebaseMLModelDownloader/Sources/ModelDownloader.swift

```diff
@@ -320,3 +329,20 @@ extension ModelDownloader {
     }
   }
 }
+
+/// Possible error messages while using model downloader.
+extension ModelDownloader {
+  private static let defaultAppNotConfiguredErrorDescription =
+    "Default Firebase app not configured."
+  private static let parseModelNameErrorDescription = { (path: String) in
+    "Unable to parse model file name at \(path)."
+  }
+
+  private static let retrieveLocalModelInfoErrorDescription =
+    "Failed to get stored model info for model file."
+  private static let outdatedModelPathErrorDescription = "Outdated model paths in local storage."
+  private static let deletedLocalModelInfoErrorDescription =
+    "Model unavailable due to deleted local model info."
+  private static let backgroundModelDownloadErrorDescription: StaticString =
```

Comment:
I'm curious whey `StaticString` here?

Reply:
So this is a `os_log` requirement which will go away when I replace it with GULLogger hopefully. Is there a better interim solution I could use?

Reply:
No, all looks good. I just was curious.